eras/alonzo/impl/CHANGELOG.md

```diff
@@ -31,10 +31,10 @@
 ## 1.6.0.0
 
 * Deprecated `getAlonzoSpendingTxIn` in favor of `plutusPurposeSpendingTxIn`
-* Add `PlutusPurpose`, `plutusPurposeSpendingTxIn` and `upgradePlutusPurposeAsIndex` to
+* Add `PlutusPurpose`, `plutusPurposeSpendingTxIn` and `upgradePlutusPurposeAsIx` to
```

Comment:
We can't change entries in the changelog for releases that has already been published, unless it's a mistake in an entry itself.
Also, since that was a rename `upgradePlutusPurposeAsIndex` -> `upgradePlutusPurposeAsIx`, it needs to be noted in the 1.7.0 section above
```suggestion
* Add `PlutusPurpose`, `plutusPurposeSpendingTxIn` and `upgradePlutusPurposeAsIndex` to
```

Reply:
Darn! I was trying to be careful about that, but for some reason I thought all the changes were above the 1.6.0.0 line.